### ISSUE TITLE
Settings UX editor: do not read "Group" for every item

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52465]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown.
3. Read all the lists,list items,buttons,links present on the screen 

**Actual:**
VoiceOver reads group for each list,listitem,buttons i.e.(Commonly Used,Editor,Workbench………..etc)

**Expected:**
VoiceOver should only read the name of the control along with relevant state,value and role. It should not only read "Group" for every control present on the screen but also its name correspondingly.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to get clarity about control's name,state,role,value and other important info if screenreader reads only as group after each word it reads.

**MAS Reference:**
MAS 1.3.1- https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_UserSettings_VoiceOver_Group.pptx](https://github.com/Microsoft/vscode/files/2122539/A11y_VSCode_UserSettings_VoiceOver_Group.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Also assigning to @roblourens so you can figure out if this is low hanging

**Comment by roblourens** [comment]
This is not the new settings editor, this is the current one.

When we tab through the section headers as shown in the ppt attachment, we're focusing view zones but it doesn't read the text. The main view zone div has `aria-hidden`, if I remove that it seems to read the text correctly.

https://github.com/Microsoft/vscode/blob/3d35801127f0a62d58d752bc613506e836c5d120/src/vs/editor/browser/viewParts/viewZones/viewZones.ts#L52

@alexandrudima do you know why the whole viewZones div is marked as `aria-hidden: true`?

**Comment by isidorn** [comment]
@roblourens my mistake, assigning to @sandy081 then also

**Comment by sandy081** [comment]
This settings editor is based on text editor. Since we are coming up with a new settings editor which is web style and will be accessible, I would not invest here.

**Comment by roblourens** [comment]
My question from above still stands but since the JSON editor is no longer the default, I'll close this.
